src/TranslatableTrait.php

```diff
@@ -0,0 +1,121 @@
+<?php
+
+namespace atk4\core;
+
+use Symfony\Contracts\Translation\TranslatorInterface;
+use Symfony\Contracts\Translation\TranslatorTrait;
+
+/**
+ * If a class use this trait, string can be translated calling method translate.
+ */
+trait TranslatableTrait
+{
+    /**
+     * Check this property to see if trait is present in the object.
+     *
+     * @var bool
+     */
+    public $_translatableTrait = true;
+
+    /**
+     * @var TranslatorInterface
+     */
+    private $translator;
```

Comment:
Still think about if this private can be raise to protected, user must use SetTranslator to set and after that is all in the hand of method _ that will check to which object delegate the translations.

Reply:
Wait,

TranslatableTrait is applied on ANY class that wants to use translations.

it adds method `_` which routes translations requests to APP or returns string as-is in case of a fallback. 

Why do we need translator defined here? Would that mean that pretty much all models and fields and whatnot will carry reference to translator?